pkgs/development/libraries/webp-pixbuf-loader/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchFromGitHub
+, meson, ninja, pkg-config
+, makeWrapper, gdk-pixbuf, libwebp
+}:
+let
+  moduleDir = gdk-pixbuf.moduleDir;
+
+  # turning lib/gdk-pixbuf-#.#/#.#.#/loaders into lib/gdk-pixbuf-#.#/#.#.#/loaders.cache
+  # removeSuffix is just in case moduleDir gets a trailing slash
+  loadersPath = (lib.strings.removeSuffix "/" gdk-pixbuf.moduleDir) + ".cache";
+in
+stdenv.mkDerivation rec {
+  pname = "webp-pixbuf-loader";
+  version = "0.0.3";
+
+  src = fetchFromGitHub {
+    owner = "aruiz";
+    repo = pname;
+    rev = version;
+    sha256 = "1dp4xrdz2p4vx2gd871vcy0nk6lkai4r3ddiw38ic6a8pq9npn11";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config ];
+  buildInputs = [ makeWrapper gdk-pixbuf libwebp ];
```

Comment:
Shall we move `makeWrapper` to `nativeBuildInputs`?

Reply:
I see, that makes sense